vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -530,26 +612,30 @@ void vtkF3DRenderer::ConfigureGridUsingCurrentActors()
         tmpUnitSquare = pow(10.0, round(log10(diag * 0.1)));
       }
 
-      double gridPos[3] = { 0, 0, 0 };
+      double center[3];
```

Comment:
Since `MultiplyDoublePoint` is expecting a `double[4]`, I think you'll have a potential out of bound issue there.  
Let's allocate `double center[4];` (and set the fourth component to 1 since it's a point, even if it won't make a difference here because there's no translation)